### PR TITLE
Merge FAC Beta 1 (final) into Master post go-live

### PR DESCRIPTION
**Pull request by JoeHS**

### CODE REVIEW

File: DFC.App.FindACourse/Controllers/CourseController.cs [hunk 1]
```diff
@@ -119,20 +123,43 @@ public IActionResult Breadcrumb()
         }
 
         [HttpGet]
-        [Route("find-a-course/search/{articleName}/body")]
+        [Route("find-a-course/search/find-courses-to-get-a-job/body")]
+        [Route("find-a-course/search/find-courses-to-get-a-job")]
```
Comment (PurpleDuckMedia) [inline DFC.App.FindACourse/Controllers/CourseController.cs:127]: @defc0nONE Hi Chris, just checking why do we need 2 different routes?

File: DFC.App.FindACourse/Controllers/CourseController.cs [hunk 2]
```diff
@@ -119,20 +123,43 @@ public IActionResult Breadcrumb()
...
+        {
+            var model = new BodyViewModel { Content = new HtmlString("Find a course: Body element") };
+            model.CourseSearchSettings = courseSearchSettings;
+            model.SideBar = GetSideBarViewModel();
+            return View("help-with-choices", model);
+        }
+
+        [HttpGet]
         [Route("find-a-course/search/body")]
+        [Route("find-a-course/search/search/body")]
```
Comment (PurpleDuckMedia) [inline DFC.App.FindACourse/Controllers/CourseController.cs:149]: @defc0nONE Hi Chris, just checking why do we need 2 different routes?

File: DFC.App.FindACourse/wwwroot/StaticRobots.txt [hunk 3]
```diff
@@ -1,3 +1,2 @@
-﻿User-agent: *
-
-Disallow: /StaticRobots.txt
+﻿User-agent: PowerMapper
```
Comment (PurpleDuckMedia) [inline DFC.App.FindACourse/wwwroot/StaticRobots.txt:1]: @defc0nONE is this an intentional change for fac beta?

**Review by PurpleDuckMedia — COMMENTED** [review]
Hi Chris, just added a few comments, can you take a look when you have a sec.

File: DFC.App.FindACourse/wwwroot/StaticRobots.txt (hunk 3 above)
Comment (defc0nONE) [reply DFC.App.FindACourse/wwwroot/StaticRobots.txt:1]: Judging from a DfE Slack conversation, this was edited to allow for a Methods consults to allow for Accessibility testing - we should revert this if that's the case

File: DFC.App.FindACourse/Controllers/CourseController.cs (hunk 1 above)
Comment (defc0nONE) [reply DFC.App.FindACourse/Controllers/CourseController.cs:127]: This is the one I'm least unsure about (we were meant to have a discussion but it never came to be) and I've had a dig around the controller - this doesn't seem to be the first instance of this (for example, the `HttpGet` route at L77-80 also has similar route duplication and can be seen in a commit [here](https://github.com/SkillsFundingAgency/dfc-app-findacourse/pull/72/commits/54d6670012abdc096c50c7039c539e4dc5f48183)) and commits merely imply about getting pages 'working' so I wonder if this is a fundamental issue with how the application is built

**Comment by defc0nONE** [comment]
@PurpleDuckMedia Sorry, not sure if my replies on both PRs notified you - just writing this filler comment in case

File: DFC.App.FindACourse/wwwroot/StaticRobots.txt (hunk 3 above)
Comment (PurpleDuckMedia) [reply DFC.App.FindACourse/wwwroot/StaticRobots.txt:1]: Actually talking to Joe. I think we can leave this as is. Doesn't cause any issues

**Review by PurpleDuckMedia — APPROVED** [review]

File: DFC.App.FindACourse/wwwroot/StaticRobots.txt (hunk 3 above)
Comment (defc0nONE) [reply DFC.App.FindACourse/wwwroot/StaticRobots.txt:1]: Sure, sounds good to me.

**Pull request merged**